src/main/java/se/simonsoft/cms/transform/rest/TransformResource.java

```diff
@@ -87,13 +83,10 @@ public Response importItem(@QueryParam("itemId") String itemIdParam, String body
                     .build();
                     
         } catch (IllegalArgumentException e) {
-            logger.warn("Invalid itemId parameter: {}", itemIdParam, e);
-            return Response.status(Response.Status.BAD_REQUEST)
-                    .entity("{\"error\": \"Invalid itemId parameter: " + e.getMessage() + "\"}")
-                    .build();
-                    
+            logger.warn("Invalid itemId parameter: {}", itemId.getLogicalId(), e);
```

Comment:
SLF will call toString on objects and itemId is good with that. Adding method call when logging should be avoided because it can cause NPEs (this one is guarded, but should be consistently avoided when possible).